autogpt_platform/backend/backend/blocks/persistence.py

```diff
@@ -0,0 +1,153 @@
+import logging
+from typing import Any, Literal
+
+from autogpt_libs.utils.cache import thread_cached
+
+from backend.data.block import Block, BlockCategory, BlockOutput, BlockSchema
+from backend.data.model import SchemaField
+
+logger = logging.getLogger(__name__)
+
+
+@thread_cached
+def get_database_manager_client():
+    from backend.executor import DatabaseManagerAsyncClient
+    from backend.util.service import get_service_client
+
+    return get_service_client(DatabaseManagerAsyncClient, health_check=False)
+
+
+def get_storage_key(key: str, scope: str, graph_id: str) -> str:
```

Comment:
since this type is used in multiple places:
```suggestion
StorageScope = Literal["within_agent", "across_agents"]


def get_storage_key(key: str, scope: StorageScope, graph_id: str) -> str:
```